python_modules/libraries/dagster-hightouch/dagster_hightouch_tests/test_placeholder.py

```diff
@@ -0,0 +1,2 @@
+def test_placeholder():
```

Comment:
Is there a reason there is a placeholder test?

Reply:
Tox was failing with a strange error (exit status 5) when there were no tests present.  🤔